deploy/certified-metadata-bundle/cockroach-operator/latest/manifests/crdb.cockroachlabs.com_crdbclusters.yaml

```diff
@@ -52,20 +52,6 @@ spec:
               dataStore:
                 description: (Required) Database disk storage configuration
                 properties:
-                  emptyDir:
```

Comment:
Is this removal expected?

Reply:
@chrisseto  we removed empty dir... it remained in one example but it will be fixed in my next PR